test/relayclient/RelayClient.test.ts

```diff
@@ -555,6 +561,73 @@ describe('RelayClient', function () {
         });
         expect(to).to.be.equal(relayHub.address);
       });
+
+      describe('with contract execution during deploy', function () {
```

Comment:
We don't need to specify the deployment since we're already in the deployment describe section
```suggestion
      describe('with contract execution', function () {
```

Reply:
updated